### DISCUSSION TITLE
Bot disconnects when I try to play something

### DISCUSSION THREAD

**Discussion by Iversthi** [jagrosh/MusicBot#1565] [Troubleshooting]
I have finally got the bot to respond, but now when I play something it just says "added xxx to begin playing and then it join the voice channel and leaves immediately.

**Comment by SzegenY** [comment]
Same issue here

**Comment by Slimee44** [comment]
Also having this issue when trying to get the bot to play my shoutcast stream... worked fine yesterday, today the bot joins the channel, does nothing, leaves, rinse/repeat

**Comment by YoneWolf** [comment]
I got the same issue here

**Comment by jagrosh** [comment]
Duplicate of several existing issues

**Reply by YoneWolf** [reply]
version 3.9 works fine
